docker/app/Dockerfile

```diff
@@ -156,7 +159,9 @@ RUN chmod +x /usr/local/bin/install_pg_repack.sh \
 
 WORKDIR /app
 
+# CVE-2025-58767, note rexml is bundled with ruby, and must be explicitly pinned
 RUN gem update --system \
+  && gem update rexml \
```

Comment:
oh hey, could we add `ripgrep` for developer QOL as long as we are tweaking the images? Not super important